src/PackedBox.php

```diff
@@ -56,6 +56,8 @@ public function getBox(): Box
 
     /**
      * Get items packed.
+     *
+     * @return PackedItemList|PackedItem[]
```

Comment:
This one I'm not sure about, that really is a `PackedItemList` and not a generic `Traversable`

Reply:
It can probably be reverted without losing any benefit since `PackedItemList#getIterator` vardocs were also updated.